src/containers/share-modal/components/ShareDrawer.module.css

```diff
@@ -3,18 +3,22 @@
   flex-direction: row;
   align-items: center;
   justify-content: center;
-  padding: 8px 0px 16px;
+  padding: var(--unit-2) 0px var(--unit-4);
 }
 
 .titleIcon {
-  height: var(--unit-6);
-  width: var(--unit-6);
+  height: var(--unit-8);
```

Comment:
this is fine since you didn't add it, but i think based on a last discussion we had as a product eng group, we're going to git rid of our 4px unit system b/c we don't adhere to it enough